wagtail/permission_policies/base.py

```diff
@@ -26,6 +26,8 @@ class BasePermissionPolicy:
     fine-grained permission logic).
     """
 
+    perm_cache_name = ""
```

Comment:
nitpick: better to write out "permission" in full rather than abbreviating it to "perm"

Reply:
Django uses `_user_perm_cache` so I followed it, but yeah I think it's better to write it in full. Fixed in e2c5a3c.